images/ci-openshift-golang-builder-latest.yml

```diff
@@ -14,7 +14,7 @@ content:
       upstream_image: registry.ci.openshift.org/ocp-multi/builder:rhel-9-golang-latest-openshift-{MAJOR}.{MINOR}
 
 from:
-  stream: rhel-9-golang
+  stream: rhel-9-golang-1.21
```

Comment:
Not sure about calling 1.21 "latest". I might turn out to be wrong, but I think 4.15 has 1.20 as its golang version.